hub/__init__.py

```diff
@@ -45,6 +45,7 @@
 delete = dataset.delete
 rename = dataset.rename
 copy = dataset.copy
+deep_copy = dataset.deep_copy
```

Comment:
shouldn't we keep the name deepcopy instead of deep_copy to make it similar to copy.deepcopy?